### ISSUE TITLE
where are the cosine softmax classifier and the corresponding loss in the code?

### ISSUE THREAD

**Issue by haochange** [nwojke/cosine_metric_learning#24]

**Comment by nwojke** [comment]
Have a look [here](https://drive.google.com/drive/folders/13HtkxD6ggcrGJLWaUcqgXl2UO6-p4PK0) for a pointer to the loss function implementation within the training code. This function itself calls [_create_softmax_loss](https://github.com/nwojke/cosine_metric_learning/blob/master/train_app.py#L612) which sets up the cross entropy loss. The actual classifier head is implemented [here](https://github.com/nwojke/cosine_metric_learning/blob/master/nets/deep_sort/network_definition.py#L86).

Sorry for the late reply, I hope it helps.

**Comment by nwojke** [comment]
I am closing this issue due to inactivity. Re-open if the problem is not solved.

**Comment by RaymondCM** [comment]
Hello @nwojke is there any reason you renamed Cross Entropy Loss to Cosine Softmax Loss?
